kge/config-default.yaml

```diff
@@ -256,6 +256,9 @@ train:
     # Keep this many most recent additional checkpoints.
     keep: 3
 
+    # keep checkpoint containing initialized model for reproducibility
```

Comment:
Rewrite to the following?

Whether to always keep checkpoint containing initialized model ('checkpoint_00000.pt') for reproducibility.

On second thoughts, I think default = true is probably a good idea.

Reply:
Other than this, the PR is ready. If it passes all test, please go ahead and update changelog and commit.